Sources/WeaverCodeGen/SwiftGenerator.swift

```diff
@@ -232,7 +232,7 @@ private final class MetaWeaverFile {
     // MARK: - Main File
 
     func meta() throws -> Meta.File {
-        return File(name: "Weaver.swift")
+        return File(name: "main-file")
```

Comment:
this is implicit place 😞 file name is not important here. This instance needed only for get `swiftString` in `generate()` function

same for tests

Reply:
I see, that's something I may have to update in Meta.